src/app/dev/DevToys.Api/Tool/GUI/Components/UIDiffListInput.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace DevToys.Api;
+
+/// <summary>
+/// Defines the different list comparaison mode
+/// </summary>
+public enum ListComparisonMode
```

Comment:
This looks again like an implementation detail of the tool you're proposing. I would suggest to move this enum to `DevToys.Tools/Models` folder.

Reply:
If I keep the UI component in "Devtoys.Api", It will create a dependance between Devtoys.Api and Devtoys.Tool.
I don't think it's desirable, what do you think ?

Reply:
As I'm suggested in my other comment to remove completely `IUIDiffListInput`, there will be no dependence needed.

Reply:
Done